Mathlib/LinearAlgebra/InvariantBasisNumber.lean

```diff
@@ -302,20 +305,28 @@ private def induced_equiv [Fintype ι'] (I : Ideal R) (e : (ι → R) ≃ₗ[R]
 
```

Comment:
There are two no longer used `private def`s above this which I'm not sure whether should be removed ...